modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -370,4 +371,6 @@ Liferay.Util.Cookie = Cookie;
 Liferay.Util.LocalStorage = localStorage;
 Liferay.Util.SessionStorage = sessionStorage;
 
+Liferay.Util.printPage = printPage;
```

Comment:
We want to actually move away from creating more utils on the global namespace. So instead I would remove this and then leverage your function via an import `import {printPage} from 'frontend-js-web`

Reply:
Hello, @bryceosterhaus This `import {printPage} from 'frontend-js-web` is used to import it in a JS file, but I want to import it in a JSP file, is there another way to import it?

Reply:
@bryceosterhaus could I use `<aui:script require="frontend-js-web/.../printPage as printPage">` to import it?

Reply:
Yeah I think you could either use that script tag, or even better would be to extract it to a JS file and use the `<liferay-frontend:component>` tag